synapse/event_auth.py

```diff
@@ -729,16 +730,36 @@ def _check_power_levels(
 ) -> None:
     user_list = event.content.get("users", {})
     # Validate users
-    for k, v in user_list.items():
+    for k in user_list.keys():
         try:
             UserID.from_string(k)
         except Exception:
             raise SynapseError(400, "Not a valid user_id: %s" % (k,))
 
-        try:
-            int(v)
-        except Exception:
-            raise SynapseError(400, "Not a valid power level: %s" % (v,))
```

Comment:
Do we want to keep this check around for older room versions?

Reply:
Yes indeed-I have fixed.